src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,73 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
         // test your code here
+        DriverService driverService
+                = (DriverService) injector.getInstance(DriverService.class);
+
+        List<Driver> bmwDrivers = new ArrayList<>();
+        Driver john = new Driver("John", "45698");
+        Driver paul = new Driver("Paul", "77777");
+        driverService.create(john);
+        driverService.create(paul);
+        bmwDrivers.add(john);
+        bmwDrivers.add(paul);
+
+        ManufacturerService manufacturerService
+                = (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        Car bmw = new Car("BMW Super", manufacturerService.get(1L), bmwDrivers);
```

Comment:
I would use an existing entity and its identifier in the database for sure, what if no such id existed? But I don't think that's a violation in this case.

Reply:
It exists. It is created by initdb file.